extensions/cdc-debezium/src/test/java/com/hazelcast/jet/cdc/AbstractCdcIntegrationTest.java

```diff
@@ -58,7 +58,7 @@
 
     @BeforeClass
     public static void beforeClassCheckDocker() {
-        assumeTrue(DockerClientFactory.instance().isDockerAvailable());
+        assumeFalse(DockerTestUtil.dockerDisabled());
```

Comment:
Could we make it `assumeTrue(DockerTestUtil.dockerEnabled());` to avoid the double negative?